Source/JavaScriptCore/runtime/IntlDurationFormat.cpp

```diff
@@ -330,8 +350,10 @@ static Vector<Element> collectElements(JSGlobalObject* globalObject, const IntlD
     auto scope = DECLARE_THROW_SCOPE(vm);
 
     bool done = false;
+    bool signDisplayed = true;
     String separator;
     Vector<Element> elements;
+    std::optional<DurationSignType> durationSign = std::nullopt;
```

Comment:
`std::nullopt` is the default value for `std::optional`, so no need to write it explicitly.
```suggestion
    std::optional<DurationSignType> durationSign;
```